bundles/pom.xml

```diff
@@ -44,6 +41,19 @@
                 <module>nightly</module>
             </modules>
         </profile>
+        <profile>
+            <id>no-deploy</id>
+            <activation>
+                <property>
+                    <name>!deploy</name>
+                </property>
+            </activation>
+            <modules>
+                <module>others</module>
+                <module>distribution</module>
```

Comment:
and what about main distribution zip - this one?

Reply:
But distribution zip is not deployed to Maven repo just only to Eclipse.org download pages. _release.sh_ creates it by `mvn --no-transfer-progress -DskipTests clean install`.

Reply:
I forgot about that... then it does not make sense to deploy rename neither, at least not yet...